tests/signals_post_constraints/statistics

```diff
@@ -0,0 +1,27 @@
+# 1: number of constraints
```

Comment:
No, of course not :-) This is the signal that is specifically triggered at this point ;-)